sympy/utilities/lambdify.py

```diff
@@ -450,6 +455,44 @@ def wrapper(*argsx, **kwargsx):
                     "Expression:\n\n{expr}").format(sig=sig, expr=expr_str)
     return func
 
+def should_dummify(expr, printer=None):
+    """
+    Returns the value of dummify to be used. This function either returns True or False.
```

Comment:
the main objective of this function is to detect keywords and python identifiers . I could have written this in the lambdify function but then for easy understanding I created a new function.